### PR TITLE
Restrict DNS

### PR DESCRIPTION
**Pull request by michalc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,13 @@ docker tag jupyterhub-registry:latest  quay.io/uktrade/jupyterhub-registry:lates
 docker push quay.io/uktrade/jupyterhub-registry:latest
 ```
 
+## DNS Server
+
+docker build -t jupyterhub-dnsmasq -f Dockerfile-dnsmasq . && \
```
Comment (lgarvey) [inline README.md]: Any benefit to making that an automated build in quay, if it isn't already?

Comment (michalc) [reply README.md]: @ejayesh Mentioned something like this as well. What are the pros/cons? So far, I've liked the fact that there are fewer steps to pushing images and releasing an update.

**Comment by lgarvey** [comment]
I don't see any cons - it'll just poll github and automatically build
images, e.g: https://quay.io/repository/uktrade/prometheus?tab=builds

**Comment by michalc** [comment]
> I don't see any cons - it'll just poll github and automatically build images, e.g: https://quay.io/repository/uktrade/prometheus?tab=builds

As discussed offline, will do this in separate ticket

**Pull request merged**